northstar-recovery.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Create Northstar Environment
+  hosts: localhost
+  gather_facts: False
+
+  tasks:
+    - name: Spin up new Mongo cluster from AMI
+      ec2:
+        key_name: dosomething-mrich
+        group: FD-Mongo
+        instance_type: t2.medium
+        image: ami-e4f6cff3
+        region: us-east-1
+        wait: yes
+        count: 3
+        vpc_subnet_id: subnet-6aacde23 #TMI VPC, Private Subnet
+      register: mongos
+
+    - name: Spin up new Northstar instance from AMI
+      ec2:
+        key_name: dosomething-mrich
+        group: web
+        instance_type: t2.small
+        image: ami-c3e5dcd4
+        region: us-east-1
+        wait: yes
+        count: 1
+        vpc_subnet_id: subnet-6aacde23 #TMI VPC, Private Subnet
+        private_ip: 172.30.8.20
```

Comment:
Possible nit-picky, but what if this server already exists? Maybe a conditional step before this action to see if IP is available, or if there's a pre-provisioned server already?